helm/dagster/values.yaml

```diff
@@ -151,6 +151,29 @@ dagsterWebserver:
   #     subPath: test_file.yaml
   volumeMounts: []
 
+  # Additional containers that should run as sidecars to the webserver. See:
+  # https://kubernetes.io/docs/concepts/workloads/pods/#how-pods-manage-multiple-containers
+  # For K8s versions after 1.29, prefer using extraPrependedInitContainers instead. See:
+  # https://kubernetes.io/docs/concepts/workloads/pods/sidecar-containers/
+  #
+  # Example:
+  # extraContainers:
+  #   - name: my-sidecar
+  #     image: busybox
+  extraContainers: []
+
+  # Additional init containers that should run before the webserver's init container, such as sidecars.
+  # For K8s versions after 1.29, see:
+  # https://kubernetes.io/docs/concepts/workloads/pods/sidecar-containers/
+  #
+  # Extra init containers are started **before** the check-db-ready init container.
```

Comment:
Unless you are reading the helm's chart implementation, which most users won't, the `check-db-ready` is unknown to readers of this configuration file.

Perhaps I'd wordsmith this as "Extra init containers are started **before** the connection to the database is tested."